### ISSUE TITLE
SH12A user1 detector has wrong id

### ISSUE THREAD

**Issue by nbassler** [DataMedSci/pymchelper#371]
should be 42, not 41.

**Comment by nbassler** [comment]
could also add future detectors which are added soon:

```C
#define SH_SDET_DOSEEQV    40  /* Dose-equivalent, ICRP 103 operational quantity. */
#define SH_SDET_EQVDOSE    41  /* Equivalent dose, ICRP 103 protection quantity. */
#define SH_SDET_USER1      42  /* User-defined detector 1. */
#define SH_SDET_USER2      43  /* User-defined detector 2. */
#define SH_SDET_NEQVDOSE   44  /* Neutron equivalent dose, ICRP 103 protection quantity. */

#define SH_SDET_Z2BETA2    45  /* z**2/beta**2 used if not averaged (e.g. for differential scorers) */
#define SH_SDET_DZ2BETA2   46  /* Dose-averaged z**2/beta**2 */
#define SH_SDET_TZ2BETA2   47  /* Track-averaged z**2/beta**2 */
#define SH_SDET_DQ         48  /* Dose-averaged Q */
#define SH_SDET_TQ         49  /* Track-averaged Q */

#define SH_SDET_Z             50  /* Charge (not averaged, e.g. for differential scorers) */
#define SH_SDET_ZEFF          51  /* Effective-charge (not averaged, e.g. for differential scorers) */
#define SH_SDET_ZEFF2BETA2    52  /* z_eff**2/beta**2 used if not averaged (e.g. for differential scorers) */
#define SH_SDET_DZEFF2BETA2   53  /* Dose-averaged z_eff**2/beta**2 */
#define SH_SDET_TZEFF2BETA2   54  /* Track-averaged z_eff**2/beta**2 */

#define SH_SDET_INVALID  32767  /* invalid detector */
```

**Comment by nbassler** [comment]
Fixed in PR #372
